src/_data/styles.json

```
@@ -6,5 +6,13 @@
     "websiteTitle": "ModernCSS.dev",
     "websiteUrl": "https://moderncss.dev",
     "stylesheet": "https://moderncss.dev/css/style.css"
+  },
+  {
+    "title": "Retroish",
+    "author": "Jean Louise Tiston",
+    "twitter": "jeanlouisedev",
+    "websiteTitle": "Jean Louise Co",
+    "websiteUrl": "http://jeanlouise.co/",
+    "stylesheet": "https://github.com/jeantiston/Retroish/blob/master/retroish.css"
```

```suggestion
    "stylesheet": "https://raw.githubusercontent.com/jeantiston/Retroish/master/retroish.css"
```

Github hosted is fine, just needs switched to the raw URL so that it's the pure stylesheet instead of the Github page :)

---

Awesome! Thanks @5t3ph